UserScrape/crawler.py

```diff
@@ -285,8 +288,9 @@ async def watch_video(self, videoId: str, main_tab: str, current_tab: str):
         duration = time_element.text
         duration = self._get_seconds(duration)
         # to make sure that every video is watched long enough
-        watch_time = duration if duration < 300 else 300 if duration/3 < 300 else duration/3
-
+        # watch_time = duration if duration < 300 else 300 if duration/3 < 300 else duration/3
+        watch_time = duration if duration < 300 else 300 
```

Comment:
We should really check whether we can afford to watch the videos for longer than 5 to 10 minutes, because if we want to repeat each condition 50 or 100 times, watching the videos will take ages

Reply:
For the training videos, we should watch them in a way that represents someone who liked the video (I can't remember what we decided). 

Confluent will allow us to run many in paralel. A linux container 1vcpu 4G mem costs 5c/hr to run on Azure Container instances.